development/dependency/python-base/Dockerfile

```diff
@@ -3,4 +3,5 @@ FROM    python:3.5.7-alpine3.8
 RUN     pip install virtualenv
 RUN     pip install flask
 RUN     pip install jsonschema
-RUN     pip install kafka-python
\ No newline at end of file
+RUN     pip install kafka-python
+RUN     pip install gunicorn
```

Comment:
@rfrsilva do we need this here?
Won't it be enough to have it in the monitor-server-python/Dockerfile ??

Reply:
Done